code/modules/spells/general/mark_recall.dm

```diff
@@ -36,7 +36,7 @@
 	var/turf/T = get_turf(target)
 	if(!T)
 		return 0
-	do_teleport(user,T)
+	user.forceMove(T)
```

Comment:
The wizard should be held to the same restrictions as other teleporting antagonists. Why is the wizard the only antag that can just warp into the vault/the armory/the bridge for free loot at round-start?

Reply:
That is not how this works, you need to reach the place where you want to set the mark, so, if you want to teleport back to the vault using this, you had to manage breaking into it first.

Reply:
PLEASE COME UP WITH A SOLUTION BETWEEN YOURSELVES